pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -200,19 +213,23 @@ func (p *staticPolicy) validateState(s state.State) error {
 		if len(tmpAssignments) != 0 {
 			return fmt.Errorf("default cpuset cannot be empty")
 		}
```

Comment:
so this branch won't be exercised anymore.

Reply:
So, while I still VERY MUCH want this cleanup, it's also true it has unexpected and deeper consequences.
Unless you are up to pursue this direction (there are options, but more code changes required), better to take it away and focus on it on a separate PR. Please undo my suggestion, let's inline (and uglify again :\) the initialization in validateState()